app/main/Mindfulness.tsx

```diff
@@ -3,39 +3,32 @@
 "use client";
 
 import '../css/Mindfulness.css'; // Import CSS file
+import { videoList } from '@/lib/data'; // Import video data
 
 interface VideoPlayerProps {
   videoId: string;
-  Label: string;
+  label: string;
 }
 
-const generateVideoPlayer = ({ videoId, Label }: VideoPlayerProps) => (
-  <div className="video-wrapper">
+const generateVideoPlayer = ({ videoId, label }: VideoPlayerProps) => (
+  <div className="video-wrapper" key={label}>
     <iframe
       className='video'
       title='Youtube player'
-      aria-label={Label}
+      aria-label={label}
       src={`https://www.youtube.com/embed/${videoId}`}
     ></iframe>
   </div>
 );
 
 function Mindfullness() {
-
-  const youtube1ID = 'AKN_gbj8UtU'
-  const youtube2ID = 'c7IkEhKtUwI'
-  const youtube3ID = 'sG69_omRCyo'
-  const youtube1Label = 'youtube-video-1';
-  const youtube2Label = 'youtube-video-2';
-  const youtube3Label = 'youtube-video-3';
-
   return (
     <div className="mindfulness-container">
       <h2 className="title">Mindfulness</h2>
       <div className="video-container">
```

Comment:
I was thinking that this "Video Container" thing might be a component, where the `videoList` can be passed as its argument.

Reply:
Yeah that's a good idea for the next PR with the components.